flight/Modules/Battery/battery.c

```diff
@@ -105,6 +105,9 @@ static void batteryTask(void * parameters)
 
 	battery_settings_updated = true;
 	bool cells_calculated = false;
+	unsigned cells;
+	FlightBatterySettingsNbCellsGet((uint8_t*)&cells);
```

Comment:
I don't think this is actually needed, it's always either detected or set to batterySettings.NbCells later.